### PR TITLE
README reorganization, new GIF and OS-specific guides

### PR DESCRIPTION
**Pull request by ryanloney**
* Remove the Beta message
* Rearranged content on README
* Added Git to getting started list
* Moved OS-specific Install Guides to wiki
* Added buttons for Install Guides, linked to corresponding wiki page
* Added Run the Notebooks section with GIF
* Replaced jupyterlab.gif with notebooks.gif (longer clip) 
* Moved OS-specific troubleshooting to wiki pages

Let me know what you think.

### CODE REVIEW

**Review by raymondlo84 — CHANGES_REQUESTED** [review]
Hi Ryan, can you run the gif with this and get it under 5MB? 
https://ezgif.com/optimize

Now it's 20+MB and that's a bit too much?

**Comment by helena-intel** [comment]
> Hi Ryan, can you run the gif with this and get it under 5MB?
> https://ezgif.com/optimize
> 
> Now it's 20+MB and that's a bit too much?

Good point. Can we make it an mp4? That's usually much smaller. Without additional context I also found it a bit confusing to see the Tensorflow Flowers notebook there, because when the gif caught my eye it just showed the output of the tensorflow training part, which could make someone think "did they just copy the tensorflow notebook?!"

File: README.md [hunk 1]
```diff
@@ -1,113 +1,43 @@
 # 📚 OpenVINO Notebooks
 
-> 🚧 Notebooks are currently in **beta**. We plan to publish a stable release this summer. Please submit [issues](https://github.com/openvinotoolkit/openvino_notebooks/issues) on GitHub, start a [discussion](https://github.com/openvinotoolkit/openvino_notebooks/discussions) or join our [Unofficial Developer Discord Server\*](https://discord.gg/xJK7SEC54c) to stay in touch.
-
-A collection of ready-to-run Python\* notebooks for learning and experimenting with OpenVINO developer tools. The notebooks are meant to provide an introduction to OpenVINO basics and teach developers how to leverage our APIs for optimized deep learning inference in their applications.
+A collection of ready-to-run Jupyter\* notebooks for learning and experimenting with the OpenVINO™ Toolkit. The notebooks provide an introduction to OpenVINO basics and teach developers how to leverage our API for optimized deep learning inference.
```
Comment (helena-intel) [inline README.md:3]: It is not yet "this summer", to me we are still in Beta. We should at least have READMEs for every notebook before we are out of beta, in my opinion.

File: README.md [hunk 2]
```diff
@@ -1,113 +1,43 @@
...
 
-The notebooks are designed to run almost anywhere &mdash; your laptop, a cloud VM, or even a Docker container. Here's what you need to get started:
+The notebooks run almost anywhere &mdash; your laptop, a cloud VM, or even a Docker container. Here's what you need to get started:
 
 - CPU (64-bit)
 - Windows\*, Linux\* or macOS\*
 - Python\* 3.6-3.8
-
-Before you proceed to the [Installation Guide](#-installation-guide), please review the detailed [System Requirements](#%EF%B8%8F-system-requirements) below.
+- Git\*
```
Comment (helena-intel) [inline README.md:12]: Can we maybe remove this section? If we're adding git here, it starts to sound like this is a complete list. We have the install/preparation guides per OS too that will tell this, and the system requirements right below this.

File: README.md [hunk 3]
```diff
@@ -119,7 +49,7 @@ To end your Jupyter session, press `Ctrl-c`. This will prompt you to `Shutdown t
 
 To deactivate your virtualenv, simply run `deactivate` from the terminal window where you activated `openvino_env`. This will deactivate your environment.
 
-To reactivate your environment, simply repeat [Step 3](#step-3-activate-the-environment) from the Install Guide. 
+To reactivate your environment, simply repeat [Step 3](#step-3-activate-the-environment) from the Install Guide.
```
Comment (helena-intel) [inline README.md:52]: It is not required to reactivate the virtual environment to run the notebooks. Should we explain this? You need to reactivate the environment if you want to install new/updated packages. But just to run the notebooks, you can just do `jupyter lab notebooks` again.

**Comment by raymondlo84** [comment]
> > Hi Ryan, can you run the gif with this and get it under 5MB?
> > https://ezgif.com/optimize
> > Now it's 20+MB and that's a bit too much?
> 
> Good point. Can we make it an mp4? That's usually much smaller. Without additional context I also found it a bit confusing to see the Tensorflow Flowers notebook there, because when the gif caught my eye it just showed the output of the tensorflow training part, which could make someone think "did they just copy the tensorflow notebook?!"

I don't think mp4 plays properly in README. The last GIF I made was about 1.8MB so it's not that bad. =)

**Review by helena-intel — APPROVED** [review]

**Pull request merged**